e1000d/src/device.rs

```diff
@@ -87,11 +108,17 @@ struct Td {
     css: u8,
     special: u16,
 }
+
+// TODO: One // line here to describe what this GROUP of constants is
+// TODO: leave one line description of these using ///
 const TD_CMD_EOP: u8 = 1;
 const TD_CMD_IFCS: u8 = 1 << 1;
 const TD_CMD_RS: u8 = 1 << 3;
 const TD_DD: u8 = 1;
 
+/// The Intel8254x device and it's associated device memory
```

Comment:
are ALL of the register writes to the device memory? That would make a lot more sense here!